gatsby-node.js

```diff
@@ -29,15 +35,15 @@ exports.sourceNodes = async ({ actions }) => {
   createTypes(typeDefs);
 
   const allSpecs =
-    // map into these results and create nodes
+    // crate endpoint smap
```

Comment:
_crate_ or _create_?

Reply:
this does look like a typo for `create endpoints map`, though maybe Alban was referring to some `crate` software I don't know about--and a `smap` software too for that matter 

I'll change this now